### PR TITLE
Properly shim exchangeratesapi

### PR DESCRIPTION
**Pull request by michel-tricot**
## What
Properly shim exchangeratesapi

### CODE REVIEW

**Review by cgardens — APPROVED** [review]

**Review by jrhizor — APPROVED** [review]

File: dataline-integrations/singer/exchangerateapi_io/source/entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      error "Unknown option: $1"
+      shift
+      ;;
+    esac
+  done
+
+  # Singer's discovery is what we currently use to check connection
+  if [ "$DISCOVER" == 1 ]; then
+    echo2 "Checking connection..."
+    echo '{"streams":[]}' > catalog.json
```
[the remark this answers is not in the document]
Comment (michel-tricot) [reply dataline-integrations/singer/exchangerateapi_io/source/entrypoint.sh]: made it better if you want to take a look at it

[the remark this answers is not in the document]
Comment (sherifnada) [reply dataline-integrations/singer/exchangerateapi_io/source/entrypoint.sh]: Isn't this only needed for destinations? Why would we shim discovery for a tap?

[the remark this answers is not in the document]
Comment (michel-tricot) [reply dataline-integrations/singer/exchangerateapi_io/source/entrypoint.sh]: you're late to the party :) I change the logic here

[the remark this answers is not in the document]
Comment (cgardens) [reply dataline-integrations/singer/exchangerateapi_io/source/entrypoint.sh]: looks great.

**Pull request merged**